iac-extensions/docker/src/test/resources/checks/SpecificVersionTagCheck/specificVersionTagCheck.dockerfile

```diff
@@ -0,0 +1,79 @@
+ARG IMAGE_NAME=my-image
+ARG IMAGE_WITH_TAG=my-image:1.2.3
+ARG LATEST_TAG=latest
+ARG NON_LATEST_TAG=1.2.3
+
+# Noncompliant@+1
+FROM --foo=bar foobar
+
+# Noncompliant@+1 {{Use a specific version tag for the image.}}
+FROM my-image
+#    ^^^^^^^^
+# Noncompliant@+1
+FROM my-image:latest
+#    ^^^^^^^^^^^^^^^
+
+# Noncompliant@+1
+FROM foobar
+# Noncompliant@+1
+FROM foobar:latest
+
+# Noncompliant@+1
+FROM --platform=foo bar
+# Noncompliant@+1
+FROM foobar AS fb
+# Noncompliant@+1
+FROM --platform=foo bar AS fb
+# Noncompliant@+1
+FROM foobar:latest AS fb
+# Noncompliant@+1
+FROM --platform=foo bar:latest
+# Noncompliant@+1
+FROM --platform bar:latest
+# Noncompliant@+1
+FROM --platform=foo bar:latest AS fb
+# Noncompliant@+1
+FROM foobar:latest as baseWithoutTag
+
+# Noncompliant@+1
+from foobar
+
+# Noncompliant@+1
+FROM ${IMAGE_NAME}:${LATEST_TAG}
+#    ^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+# Noncompliant@+1
+FROM ${IMAGE_NAME}
+#    ^^^^^^^^^^^^^
+
+# Noncompliant@+2
+FROM \
+foobar
+
+# Noncompliant@+2
+FROM \
+foobar:latest
+
+# Compliant
+FROM my-image:1.2.3 as baseWithTag
+FROM my-image:1.2.3-alpine
+FROM ubuntu:18.04 as local-toolchain-ubuntu18.04-manylinux2010
+FROM foobar@12313423 # Compliant usage of digest
```

Comment:
[Dockerfile comments cannot be put after an instruction](https://docs.docker.com/engine/reference/builder/#format), they must be alone on their line.

> Docker treats lines that begin with # as a comment, unless the line is a valid [parser directive](https://docs.docker.com/engine/reference/builder/#parser-directives). A # marker anywhere else in a line is treated as an argument.

Reply:
Fair point, the comments should not be on the same level here